### ISSUE TITLE
Convert to script tool/python toolbox

### ISSUE THREAD

**Issue by jacobdadams** [agrc/usps-county-project#6]
Makes more sense as a GUI tool than a CLI since you'll probably already be in ArcMap/Pro to create the source addresses feature class.

**Comment by steveoh** [comment]
While this might be true, didn't you mention there is a where clause to only pull the county in question? Wouldn't it be simpler to grab the latest address points without having to filter and save?

**Comment by jacobdadams** [comment]
Good point. However, there is still some confusion as to whether we should be sending all the points for a given county or be trying to subset them somehow. Currently, I'm doing a manual review of the county before running the script.
